issue.go

```diff
@@ -200,3 +201,9 @@ func NewIssue(ctx *Context, node ast.Node, ruleID, desc string, severity Score,
 		Cwe:        GetCweByRule(ruleID),
 	}
 }
+
+// WithSuppressions set the suppressions of the issue
+func (i *Issue) WithSuppressions(suppressions []SuppressionInfo) *Issue {
```

Comment:
nit: here you don't need to return an issue pointer back since the pointer is already mutable unless you want to have this kind of style in the API when chaining multiple methods calls. What is the intend here?

Reply:
The original idea was to follow the code style of gosec. I saw all `With-` methods return pointers.